scripts/pushit.sh

```diff
@@ -67,6 +64,7 @@ case "$branch" in
     ;;
 esac
 
+dir=data
```

Comment:
because we are using a reusable workflow, we are now assuming that the branch is stored in "data"